lib/web_ui/lib/src/engine/dom_renderer.dart

```diff
@@ -509,11 +513,21 @@ flt-glass-pane * {
       });
     }
 
-    if (useCanvasKit) {
+    // Only reset CanvasKit if it wasn't already loaded.
```

Comment:
nit: not all environments will be loading it (e.g. Cobalt). I'd say "if it's not already available".

Reply:
Done.